WordPress/gutenberg
# Fix zoom out mode background color on Safari

## What?


Partially addresses https://github.com/WordPress/gutenberg/issues/60601

This fixes the background color on Safari. Like @glendaviesnz correctly [points out](https://github.com/WordPress/gutenberg/issues/60601#issuecomment-2046203183) Safari is not rendering the background color on the area occupied by the element before the transform. The way we were styling it before was working because other browsers weren't behaving as intended.

## Why?


It was a bug

## How?


By changing the color of the iframe to be gray instead of white.

## Testing Instructions





- Activate the zoom-out experiment
- On the site editor, click on the zoom-out icon, the background should appear gray in all browsers. 


### Testing Instructions for Keyboard


## Screenshots or screencast 

Before (Safari):

<img width="1456" alt="Screenshot 2024-04-18 at 17 48 26" src="https://github.com/WordPress/gutenberg/assets/3593343/7df80e0d-a0ad-4062-8da3-842db4bf04de">

After (Safari):

<img width="1457" alt="Screenshot 2024-04-18 at 17 48 08" src="https://github.com/WordPress/gutenberg/assets/3593343/5940179a-dbd4-4afa-9984-4832fdf1ad7d">
